.github/workflows/ci-build.yml

```diff
@@ -19,47 +18,41 @@ jobs:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
```

Comment:
why not sync up with blueprint to use `pnpm/action-setup` and `actions/setup-node`?

Reply:
less to think about, tbh.
also also back when I made action-setup-pnpm, both setup-node and pnpm/action-setup did not infer either of their versions from the correct places. 
_Today_ the difference is (per job) 1 line vs ~ 4, I think, not too bad